### ISSUE TITLE
The communication server encountered the following error on install :  Err_02

### ISSUE THREAD

**Issue by nohnaimer** [OCSInventory-NG/OCSInventory-Server#48]
On install Plugins take this error The communication server encountered the following error on install :  Err_02

### General informations
Operating system : 
CentOS 7.3 x64
### Server informations
Perl version : 5.16.3
Mysql / Mariadb / Percona version :  
Percona-Server 5.7.17-11
### OCS Inventory informations
Ocs server version : 
2.3
### Problem's description
On error log httpd server

> [Fri Mar 03 15:11:02.809043 2017] [php7:warn] [pid 7061] [client 10.128.1.62:60925] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197
[Fri Mar 03 15:11:18.251413 2017] [php7:warn] [pid 7077] [client 10.128.1.62:60941] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197
[Fri Mar 03 15:11:21.188128 2017] [php7:warn] [pid 7077] [client 10.128.1.62:60941] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/
[Fri Mar 03 15:11:25.409652 2017] [php7:warn] [pid 7077] [client 10.128.1.62:60941] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/
[Fri Mar 03 15:11:36.042489 2017] [php7:warn] [pid 7064] [client 10.128.1.62:60957] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/
[Fri Mar 03 15:11:36.862132 2017] [php7:warn] [pid 7064] [client 10.128.1.62:60957] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/
[Fri Mar 03 15:11:38.063886 2017] [php7:warn] [pid 7064] [client 10.128.1.62:60957] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/
[Fri Mar 03 15:11:38.547069 2017] [php7:warn] [pid 7064] [client 10.128.1.62:60957] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/
[Fri Mar 03 15:11:39.741255 2017] [php7:warn] [pid 7064] [client 10.128.1.62:60957] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/
[Fri Mar 03 15:11:40.908124 2017] [php7:warn] [pid 7064] [client 10.128.1.62:60957] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/
[Fri Mar 03 15:11:41.047855 2017] [php7:warn] [pid 7064] [client 10.128.1.62:60957] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/?function=visu_computers
[Fri Mar 03 15:11:45.801414 2017] [php7:warn] [pid 7064] [client 10.128.1.62:60957] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/?function=visu_computers
[Fri Mar 03 15:16:30.621887 2017] [php7:warn] [pid 7079] [client 10.128.1.62:61245] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/index.php?function=account_config&head=1
[Fri Mar 03 15:16:30.758288 2017] [php7:warn] [pid 7079] [client 10.128.1.62:61245] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/index.php?function=pluginsmanager
[Fri Mar 03 15:16:32.811900 2017] [php7:warn] [pid 7079] [client 10.128.1.62:61245] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/index.php?function=account_config&head=1
[Fri Mar 03 15:16:32.929047 2017] [php7:warn] [pid 7079] [client 10.128.1.62:61245] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/index.php?function=pluginsmanager
[Fri Mar 03 15:16:38.176266 2017] [php7:warn] [pid 7064] [client 10.128.1.62:61264] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/index.php?function=pluginsmanager
[Fri Mar 03 15:16:38.468429 2017] [php7:warn] [pid 7064] [client 10.128.1.62:61264] PHP Warning:  mysqli_fetch_row() expects parameter 1 to be mysqli_result, boolean given in /usr/share/ocsinventory-reports/ocsreports/require/header.php on line 197, referer: http://ocs.inside.mts.by/index.php?function=pluginsmanager

Try to reinstall, same problem. Try to install last committed ocs inventory server and ocsreports - same error!
Any idea?

**Comment by gillesdubois** [comment]
Hi,

It's on plugin related issues. it's been updated in the trunk ocs. A new version will be available soon.

Regards,
GIlles Dubois.

**Comment by Adrianodl** [comment]
Hi, found the same problem here, but I think I have a solution. =')
Checking my Apache Logs I get this message:
PHP Warning:  fgets() expects parameter 1 to be resource, boolean given in /usr/share/ocsinventory-reports/ocsreports/plugins/main_sections/ms_plugins/plugins.class.php on line 261, referer: http://XXX.XXX.XXX.XXX/ocsreports/index.php?function=pluginsmanager
Reading plugins.class.php I've found the variable $reading (oh, the irony!) receive language/ENGLISH/ENGLISH.txt (the uppercase are mine to enhance visualisation) BUT inside the directory language in the version 2.4 of OCS server the english language lies on en_GB!
So I've changed where I found as english to en_GB.
Using vim just use the comand:  :%s/english/en_GB/g
After this modification in the plugins.class.php file, the installation of the plugin worked as well.
I hope it helps, sorry for my poor english skills. (Hehe!)
